crates/storage/src/collections/hashmap/fuzz_tests.rs

```diff
@@ -172,3 +185,80 @@ fn fuzz_defrag(xs: Vec<i32>, inserts_each: u8) {
     })
     .unwrap()
 }
+
+impl<
+        K: Arbitrary + Ord + PackedLayout + Send + Clone + std::hash::Hash + 'static,
+        V: Arbitrary + PackedLayout + Send + Clone + 'static,
+    > Arbitrary for StorageHashMap<K, V>
+{
+    fn arbitrary(g: &mut Gen) -> StorageHashMap<K, V> {
+        let hmap = HashMap::<K, V>::arbitrary(g);
+        StorageHashMap::<K, V>::from_iter(hmap.into_iter())
```

Comment:
```suggestion
        StorageHashMap::<K, V>::from_iter(hmap)
```

Reply:
note that I am not entirely sure with this. please merge my suggestion if it works. but iirc `from_iter` take its input as `T: IntoIter`.